app/javascript/application.js

```diff
@@ -9,3 +9,23 @@ import "@rails/actiontext"
 document.addEventListener('turbo:load', () => {
   createApp(ProjectsComponent).mount('#vue-projects-app')
 })
```

Comment:
Aproveita e adiciona essa linha de código logo abaixo da linha 9, pois esse código tá tentando montar esse componente em qualquer rota, prejudicando a performance do nosso app e fazendo computações desncessárias.
```suggestion
document.addEventListener('turbo:load', () => {
  if (window.location.pathname === '/projects') {
    createApp(ProjectsComponent).mount('#vue-projects-app')
  }
})
```